wct-core/src/main/java/org/webcurator/ui/target/validator/ProfilesOverridesValidator.java

```diff
@@ -65,8 +65,11 @@ public void validate(Object comm, Errors errors) {
 			} else {
 				log.info("Skipping imported profile validation as there is no profile override.");
 			}
-		} else {
-			log.info("Validating non-imported profile.");
+		}
+		// Only validate Heritrix 1 fields if a Heritrix 1 profile is selected.
+		// (prevents hidden H1 fields failing validation when using an H3 profile)
+		else if (command.getHarvesterType().equals("HERITRIX1")){
+			log.info("Validating non-imported Heritrix 1.x profile.");
```

Comment:
I try to put as much information as possible in log messages as it might be useful later on when tracking down issues/debugging. Is there any other information that we can include in these additional log messages? Would information about the `command` be helpful?